Mod Menu Helper/assets/modmenu/lang/ko_kr.json

```diff
@@ -0,0 +1,55 @@
+{
+    "modmenu.dropInfo.line1": "FO 모드는 §9✎§r 또는 §8✎§r, 대체제는 §c*§r로 표시됩니다",
+    "modmenu.dropInfo.line2": "다른 모드는 지원되지 않습니다",
+    "modmenu.summaryTranslation.slight-gui-modifications": "애니메이션, 메뉴의 모드팩 버전\n§9✎§8 애니메이션, 색, FPS 인디케이터, ...",
+    "modmenu.summaryTranslation.advancementinfo": "커지고 스마트해진 도전과제 창\n§9✎§8 색, 정보 칸 크기, ...",
+    "modmenu.summaryTranslation.advancements-enlarger": "§c*§r 커진 도전과제 창\n§8✎ (옵션 없음)",
+    "modmenu.summaryTranslation.animatica": "Optifine의 애니메이션 텍스쳐\n§8✎ (고급 옵션만 있음)",
+    "modmenu.summaryTranslation.antighost": "/ghost 명령어로 Ghost 블록 제거\n§9✎§8 키바인드",
```

Comment:
You might have misunderstood what "ghost blocks" are, so check this video: https://www.youtube.com/watch?v=mHhuU9j4ORo